.github/workflows/integration_models.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: integration tests (models)
+
+on:
+  pull_request:
+    branches:
+      - trunk
+      - release-*
+      - feature-*
+    paths-ignore:
+      - 'docs/**'
+      - 'README.md'
+      - 'version.txt'
+      - 'acknowledgements.md'
+
+  workflow_dispatch:
+
+concurrency:
+  # Allow only one workflow per any non-trunk branch.
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.ref_name == 'trunk' && github.sha || 'any-sha' }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build Test Binary
+    runs-on: spiceai-runners
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          os: 'linux'
+
+      - name: Set up make
+        uses: ./.github/actions/setup-make
+
+      - name: Set up cc
+        uses: ./.github/actions/setup-cc
+
+      # Build the test binary without running tests
+      - name: Build AI integration test binary
+        run: |
+          TEST_BINARY_PATH=$(cargo test -p runtime --test integration_models --features models --no-run --message-format=json | jq -r 'select(.reason == "compiler-artifact" and (.target.kind | contains(["test"])) and .executable != null) | .executable')
```

Comment:
We should make this a Makefile target/command.

Reply:
We have the following Make command, the arguments are slightly different (--nocapture, --no-run --message-format=json). I think this way it is more visually easy to understand the command instead of wrapping it in make file - I'll review how to re-use the same/single make command and handle the args difference

```make
.PHONY: test-integration-models
test-integration-models:
	@cargo test -p runtime --test integration_models --features models -- --nocapture

.PHONY: test-integration-models-without-openai
test-integration-models-without-openai:
	@cargo test -p runtime --test integration_models --features models -- --nocapture --skip openai_test
```